app/src/main/java/de/danoeh/antennapod/fragment/FeedInfoFragment.java

```diff
@@ -168,12 +168,8 @@ private void showFeed() {
         txtvTitle.setText(feed.getTitle());
 
         String description = feed.getDescription();
-        if(description != null) {
-            if(Feed.TYPE_ATOM1.equals(feed.getType())) {
-                HtmlToPlainText formatter = new HtmlToPlainText();
-                Document feedDescription = Jsoup.parse(feed.getDescription());
-                description = StringUtils.trim(formatter.getPlainText(feedDescription));
-            }
+        if (description != null) {
```

Comment:
This null check is just present in one location. The two other files don't have it. I think the check should probably either be in all files or ditectly in HtmlToPlainText.